client/slack/member_history_chart.js

```diff
@@ -0,0 +1,65 @@
+import $ from 'jquery';
+import Chart from 'chart.js';
+
+
+function buildMemHisChart(slackMemberMonthlyCount) {
+    const obj = slackMemberMonthlyCount;
+    const memHisChart = new Chart($('#member_history_chart'), {   // eslint-disable-line
```

Comment:
If you need to disable a rule, disable it explicitly, not eslint as a whole. This is also a consideration for overriding AirBnB rules (they are very opinionated and, IMO, overly restrictive.

Also, a more descriptive variable name, like `memberHistoryChart` would be better, as the shortening `mem` can be ambiguous.